quinn-proto/src/connection/mod.rs

```diff
@@ -3140,6 +3241,19 @@ impl Connection {
                 buf.write(frame::FrameType::PATH_RESPONSE);
                 buf.write(token);
                 self.stats.frame_tx.path_response += 1;
+
+                // NOTE: this is technically not required but might be useful to ride the
+                // request/response nature of path challenges to refresh an observation
```

Comment:
When is refreshing an observation useful, given that we don't believe the path has changed?